packages/widget/src/components/Modal.tsx

```diff
@@ -172,7 +172,6 @@ const StyledOverlay = styled(Dialog.Overlay)<{
   bottom: 0;
   display: grid;
   place-items: center;
-  overflow-y: auto;
```

Comment:
This solved the rounded corners issue and didnt find it broke anythign else, but lmk if this feels important to maintain. @toddkao

Reply:
looks good